### PR TITLE
update LogIn and SignUp component

### PR DESCRIPTION
**Pull request by nuelscode**

### CODE REVIEW

File: src/components/auth/LogIn.js [hunk 1]
```diff
@@ -3,7 +3,47 @@ import Form from 'react-bootstrap/Form';
...
 import styled from 'styled-components';
 
-export const LogIn = props => {
+export class LogIn extends Component {
+    state = {
+
+    }
+    handleChange (e) {
+        console.log(e)
+    }
```
Comment (EngrEric) [inline src/components/auth/LogIn.js:12]: where are you using this func

File: src/components/auth/LogIn.js [hunk 2]
```diff
@@ -3,7 +3,47 @@ import Form from 'react-bootstrap/Form';
 import Button from 'react-bootstrap/Button';
 import styled from 'styled-components';
 
-export const LogIn = props => {
+export class LogIn extends Component {
+    state = {
+
+    }
```
Comment (EngrEric) [inline src/components/auth/LogIn.js:9]: why is your state empty,
Use the state to hold the data as the user types

Comment (EngrEric) [reply src/components/auth/LogIn.js:9]: On submit send the data on state to an API for the signup

Comment (nuelscode) [reply src/components/auth/LogIn.js:9]: Alright, on it

**Pull request merged**